pkg/validate/validate.go

```diff
@@ -48,7 +48,7 @@ func (v *HostnameValidator) Validate(input string) error {
 	}
 
 	// check for special characters
-	if strings.ContainsAny(input, "!@#$%^&*()") {
+	if strings.ContainsAny(input, "!@#$%^&*(),") {
```

Comment:
if we move `multipleHostnameValidator` into `validate` we should leave this as is an create a validator for multiple hostnames that accepts commas.

Reply:
The restrictions on domain names are pretty stringent. You can have letters (a-z, A-Z), numbers (0-9) and - plus the . that separates the segments ([rfc 1034](https://tools.ietf.org/html/rfc1034)). What's the rational for the current set of special characters to exclude?

Reply:
I left the `,` in, because otherwise `foo.nitro,bar.nitro` will pass the hostname validator and be treated as a single hostname, which I doubt very much is what is wanted.